app/views/idv/inherited_proofing/get_started.html.erb

```diff
@@ -0,0 +1,38 @@
+<% title t('inherited_proofing.headings.welcome') %>
+
+<%= render JavascriptRequiredComponent.new(
+      header: t('idv.welcome.no_js_header'),
+      intro: t('idv.welcome.no_js_intro', sp_name: decorated_session.sp_name || t('inherited_proofing.info.no_sp_name')),
+    ) do %>
+  <%= render PageHeadingComponent.new.with_content(t('inherited_proofing.headings.welcome')) %>
+  <p>
+    <%= t('inherited_proofing.info.welcome_html', sp_name: decorated_session.sp_name || t('inherited_proofing.info.no_sp_name')) %>
+  </p>
+  <h2><%= t('inherited_proofing.instructions.welcome') %></h2>
+
+  <%= render ProcessListComponent.new(heading_level: :h3, class: 'margin-y-3') do |c| %>
+    <%= c.item(heading: t('inherited_proofing.instructions.bullet1')) %>
+  <% end %>
+
+  <%= simple_form_for :inherited_proofing,
+                      url: url_for,
+                      method: 'put',
+                      html: { autocomplete: 'off', class: 'margin-y-5 js-consent-continue-form' } do |f| %>
```

Comment:
The `js-consent-continue-form` class is only relevant for the unsupervised proofing "Consent" screen, so I wouldn't expect to see it here.

```suggestion
                      html: { autocomplete: 'off', class: 'margin-y-5' } do |f| %>
```